runtime/altair/src/lib.rs

```diff
@@ -1831,6 +1832,14 @@ impl pallet_liquidity_pools_gateway::Config for Runtime {
 	type WeightInfo = ();
 }
 
+impl pallet_liquidity_pools_gateway_queue::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type Message = pallet_liquidity_pools::Message;
+	type MessageNonce = LPGatewayQueueMessageNonce;
+	type MessageProcessor = LiquidityPoolsGateway;
+	type WeightInfo = ();
```

Comment:
TODO We should add the generated weights here

Reply:
Done.